src/System.Linq/tests/SkipWhileTests.cs

```diff
@@ -165,7 +165,7 @@ public int Current
         }
 
         [Fact]
-        [OuterLoop]
+        [ActiveIssue("Valid test but too intensive to enable even in OuterLoop")]
```

Comment:
Would `[Fact(Skip="…` be more appropriate when there isn't an active issue as such, but there is still a good reason to skip it.